Dockerfile

```diff
@@ -1,6 +1,8 @@
 # This is totally hacked from the circleci container builds...
 
-FROM node:6.11
+# Node 8 started LTS on 2017-10-31
```

Comment:
YES.

Should we upgrade our .tool-versions in admin-react?

Reply:
You can upgrade once this is merged and your dockerfiles are updated (so that it would accurately reflect production).

Reply:
cc @tylerferguson who loves upgrading things